hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/KeyLifecycleService.java

```diff
@@ -219,12 +221,26 @@ public void resume() {
     suspended.set(false);
   }
 
+  public void setServiceEnabled(boolean enabled) {
+    this.isServiceEnabled.set(enabled);
+    LOG.info("KeyLifecycleService is {}", enabled ? "enabled" : "disabled");
+  }
+
   @Override
   public void shutdown() {
     super.shutdown();
     KeyLifecycleServiceMetrics.unregister();
   }
 
+  public GetLifecycleServiceStatusResponse status() {
+    Set<String> runningBuckets = new HashSet<>(inFlight.keySet());
+    return GetLifecycleServiceStatusResponse.newBuilder()
+        .setIsRunning(!runningBuckets.isEmpty())
+        .setIsEnabled(isServiceEnabled.get())
```

Comment:
Can we combine the IsRunning and IsEnabled to one state.  If IsRuning is the result of whether there is bucket task running or not, this info is already covered by RunningBuckets fields.

Reply:
Removed the setIsRunning.